src/views/TabView/TabBarTop.js

```diff
@@ -75,7 +75,8 @@ export default class TabBarTop extends PureComponent<DefaultProps, Props, void>
       inputRange,
       outputRange,
     });
-    const label = this.props.getLabelText(scene);
+    
+    const label = this.props.renderLabel(scene);
```

Comment:
I think probably change function name to `getLabel`, because `renderX` is supposed to return a react element.